### PR TITLE
fix(ui links): expose full path to UI

### PR DESCRIPTION
**Pull request by nwhite222**
NR-129170: Replaces the former practice of showing UI links that skip across multiple navigation nodes in the UI; updated UI path entry in the style guide to reflect this preference

### CODE REVIEW

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/introduction-service-maps.mdx [hunk 1]
```diff
@@ -59,7 +59,7 @@ Check out [How to use service maps](/docs/understand-dependencies/understand-sys
 
 <img
   title="Omnimap-powered service maps experience"
-  alt="A screenshot of the omnimap-powered maps experience. The screenshot shows a UI separated by white lines into three tiers. They are ordered vertically with UX as the top tier, Service as the second tier, and Infrastructure as the third tier."
+  al[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >eenshot shows a UI separated by white lines into three tiers. They are ordered vertically with UX as the top tier, Service as the second tier, and Infrastructure as the third tier."
```
Comment (nwhite222) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/introduction-service-maps.mdx:62]:
```suggestion
alt="A screenshot of the omnimap-powered maps experience. The screenshot shows a UI separated by white lines into three tiers. They are ordered vertically with UX as the top tier, Service as the second tier, and Infrastructure as the third tier."
```

File: src/content/docs/apm/agents/manage-apm-agents/app-naming/name-your-application.mdx [hunk 2]
```diff
@@ -201,7 +201,7 @@ The application name you set in the New Relic user interface is a human-readable
 
 Initially, this alias is the same as the app's identifier name in the New Relic agent configuration file. However, changing the UI alias **does not** affect the underlying app name used for data rollup, which is set in the agent config file. The application must be actively reporting data to change the alias. Changing the UI alias will cause all agents that report under the app identifier to restart themselves.
 
-To change the **alias** for your application's name in the New Relic user interface:
+To change t[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >interface:
```
Comment (nwhite222) [inline src/content/docs/apm/agents/manage-apm-agents/app-naming/name-your-application.mdx:204]:
```suggestion
To change the **alias** for your application's name in the New Relic user interface:
```

File: src/content/docs/apm/transactions/transaction-traces/introduction-transaction-traces.mdx [hunk 3]
```diff
@@ -81,8 +81,8 @@ To configure or edit trace settings, see the procedures for:
 To view transaction traces:
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **Transactions**.
-2. In the **Transaction traces** section, click a transaction trace to view [additional details](/docs/apm/transactions/transaction-traces/transaction-traces-summary-page).
-
+2. In the **Transaction traces** [one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >(/docs/apm/transactions/transaction-traces/transaction-traces-summary-page).
+[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/apm/transactions/transaction-traces/introduction-transaction-traces.mdx:85]:
```suggestion
2. In the **Transaction traces** section, click a transaction trace to view [additional details](/docs/apm/transactions/transaction-traces/transaction-traces-summary-page).
```

File: src/content/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.mdx [hunk 4]
```diff
@@ -42,7 +42,7 @@ Improving on traditional industry standards for measuring page load timing, we g
   alt="browser_SPA.png"
   src={browserSpaPageview}
 />
-
+[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.mdx:45]:
```suggestion
```

File: src/content/docs/apm/apm-ui-pages/monitoring/view-slow-query-details.mdx [hunk 5]
```diff
@@ -38,8 +38,8 @@ When [transaction traces](/docs/apm/transactions/transaction-traces/introduction
 To see your slow query data:
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **Databases**.
-2. Select a database transaction.
-3. If available, select any available [slow queries](#slowsql_details) listed on the page.
+2. Select a database transaction.[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
+3. If available, s[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >the page.
```
Comment (nwhite222) [inline src/content/docs/apm/apm-ui-pages/monitoring/view-slow-query-details.mdx:42]:
```suggestion
2. Select a database transaction.
3. If available, select any available [slow queries](#slowsql_details) listed on the page.
```

File: src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx [hunk 6]
```diff
@@ -19,8 +19,8 @@ The UI for external services is a great place to analyze a single service along
 
 The external services feature is available from the left navigation pane once you select an APM-monitored services. You can open it by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **External services**.
 
-## The external services map [#use-the-map]
-
+## The external services map [[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
+[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx:23]:
```suggestion
## The external services map [#use-the-map]
```

File: src/content/docs/apm/agents/java-agent/features/real-time-profiling-java-using-jfr-metrics.mdx [hunk 7]
```diff
@@ -81,7 +81,7 @@ To make troubleshooting easier, you need to see the high fidelity runtime charac
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services > (select service) > Realtime Profiling Java**: The JVM cluster timeline view shows the JVM behavior across the cluster.
+  **[[one.newrelic.com > All capabilities]](https://one.newrelic.com/all-capabilities) > APM & Services > (select service) > Realtime Profiling Java**: The JVM cluster timeline view shows the JVM behavior across the cluster.
```
Comment (nwhite222) [inline src/content/docs/apm/agents/java-agent/features/real-time-profiling-java-using-jfr-metrics.mdx:84]:
```suggestion
  **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM & Services > (select service) > Realtime Profiling Java**: The JVM cluster timeline view shows the JVM behavior across the cluster.
```

File: src/content/docs/apm/transactions/intro-transactions/monitor-background-processes-other-non-web-transactions.mdx [hunk 8]
```diff
@@ -145,5 +145,6 @@ To create new non-web transactions, follow the procedures for your APM language
 
 To view non-web transaction data in the New Relic UI, go to the main chart on the following pages, and then select the **Non-web** option in the transaction type dropdown:
 
-* APM: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & services** > (select an app) > **Summary**.
-* Transactions: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **Transactions**.
+* APM: Go to **[one.newrelic.c[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >** > (select an app) > **Summary**.
+* Transactions:[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >lities) > APM & Services**. Select an app, then under the **Monitor** section, click **Transactions**.
+[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/apm/transactions/intro-transactions/monitor-background-processes-other-non-web-transactions.mdx:150]:
```suggestion
* APM: Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM & services** > (select an app) > **Summary**.
* Transactions: Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **Transactions**.
```

File: src/content/docs/apm/transactions/transaction-traces/introduction-transaction-traces.mdx [hunk 9]
```diff
@@ -82,7 +82,6 @@ To view transaction traces:
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **Transactions**.
 2. In the **Transaction traces** section, click a transaction trace to view [additional details](/docs/apm/transactions/transaction-traces/transaction-traces-summary-page).
-
 ## Examine logs for trace details [#logs-context]
```
Comment (nwhite222) [inline src/content/docs/apm/transactions/transaction-traces/introduction-transaction-traces.mdx:85]:
```suggestion

## Examine logs for trace details [#logs-context]
```

File: src/content/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.mdx [hunk 10]
```diff
@@ -42,7 +42,6 @@ Improving on traditional industry standards for measuring page load timing, we g
   alt="browser_SPA.png"
   src={browserSpaPageview}
 />
-
 <figcaption>
```
Comment (nwhite222) [inline src/content/docs/browser/single-page-app-monitoring/get-started/introduction-single-page-app-monitoring.mdx:45]:
```suggestion

<figcaption>
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 11]
```diff
@@ -243,7 +243,7 @@ If you've enabled the [Kubernetes events integration](/docs/integrations/kuberne
 
 ## Use the cluster explorer [#cluster-explorer-use]
 
-<Callout title="Deprecated">
+<Cal[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:246]:
```suggestion
<Callout title="Deprecated">
```

File: src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx [hunk 12]
```diff
@@ -20,7 +20,6 @@ The UI for external services is a great place to analyze a single service along
 The external services feature is available from the left navigation pane once you select an APM-monitored services. You can open it by going to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & Services**. Select an app, then under the **Monitor** section, click **External services**.
 
 ## The external services map [#use-the-map]
```
Comment (nwhite222) [inline src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-ui.mdx:22]:
```suggestion
## The external services map [#use-the-map]

```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 13]
```diff
@@ -291,7 +291,7 @@ Click a node to see the following data:
 
 * Pod statistics
 * CPU, memory, and storage consumption against allocatable amounts
-* Amount of pods used by the node against the allocatable amount of pods
+* Am[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > pods
```
Comment (nwhite222) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:294]:
```suggestion
* Amount of pods used by the node against the allocatable amount of pods
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 14]
```diff
@@ -261,7 +261,7 @@ The cluster explorer represents your most relevant cluster data on a chart with
 
 * Outer ring: Contains **up to 24 nodes** of your cluster, the most relevant based on the amount of alerts. Hover over each node to check resource consumption and the percentage of allocable pods used.
 
-<img
+<img[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:264]:
```suggestion
<img
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 15]
```diff
@@ -327,10 +327,11 @@ You can also change the time frame using the time picker in the upper right corn
 
 <img
   title="New Relic - Cluster explorer - Time picker and Auto-refresh"
-  alt="New Relic - Cluster explorer - Time picker and Auto-refresh"
+  al[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:330]:
```suggestion
  alt="New Relic - Cluster explorer - Time picker and Auto-refresh"
```

File: src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx [hunk 16]
```diff
@@ -327,10 +327,11 @@ You can also change the time frame using the time picker in the upper right corn
...
   src={kubernetesClusterExplorerPickerRefresh}
 />
 
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Kubernetes cluster explorer**: The time picker lets you select several predefined time spans. To reload the data every minute, check the auto-refresh box.
 </figcaption>
+[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
```
Comment (nwhite222) [inline src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx:337]:
```suggestion
```

**Review by jeff-colucci — APPROVED** [review]
Had to make a change to one topic in light of some hero work I did yesterday, but everything else looked good to me!

**Pull request merged**